Makefile

```diff
@@ -8,6 +8,7 @@ PROJECTS_PYCODESTYLE="sdk cli toolchain"
 BUILD_PARAMS=
 LEGION_VERSION=0.11.0
 SANDBOX_PYTHON_TOOLCHAIN_IMAGE=
+LEGIONCTL_NONINTERACTIVE=1
 CREDENTIAL_SECRETS=.secrets.yaml
```

Comment:
`127.0.0.1 - - [21/Aug/2019 15:41:40] "GET /oauth/callback?state=oqkBZckwsD&session_state=7d417154-6f98-&code=cbf151ce23017f.7d417154-6f98-41
ad HTTP/1.1" 200 -`
I get this message in output when I try to log in. Please show it only when verbose is enabled.

Reply:
done